deploy/README.md

```diff
@@ -8,79 +8,207 @@ npm run compile
 2. create `.env` file
 `cp .env.example .env`
 
-3. if it is necessary to have the bridge to relay native coins to an erc20 token, the following example could be used for `.env` file
+3. If it is necessary, deploy and configure a multi-sig wallet contract which will
+   be used to manage the bridge contracts after deployment.
+
+4. adjust parameters in `.env` file as per recommendation below depending on the desired bridge mode 
+
+5. Fill balance of the deployment account in Home and Foreign networks 
+
+6. Run `node deploy.js` 
+
+## Configuration for `NATIVE-TO-ERC` Bridge mode  
+
+Here is an example of `.env` file for `native-to-erc` bridge mode.
```

Comment:
I think this information would be easier to view in a table format, similar to what exists currently on the BridgeUI site. I can make a suggestion to convert in my PR if you would prefer to not change at the moment.

Reply:
I am not sure how to address correctly the fact that some parameters have different meaning for different bridge modes.